[2024-06-26T10:47:02.162+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-06-26T10:47:02.200+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: demo_dag_for_alerting_apis_final.check_previous_run_status scheduled__2024-06-26T10:46:00+00:00 [queued]>
[2024-06-26T10:47:02.216+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: demo_dag_for_alerting_apis_final.check_previous_run_status scheduled__2024-06-26T10:46:00+00:00 [queued]>
[2024-06-26T10:47:02.216+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 6
[2024-06-26T10:47:02.244+0000] {taskinstance.py:2330} INFO - Executing <Task(BranchPythonOperator): check_previous_run_status> on 2024-06-26 10:46:00+00:00
[2024-06-26T10:47:02.252+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=6991) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-06-26T10:47:02.254+0000] {standard_task_runner.py:63} INFO - Started process 6992 to run task
[2024-06-26T10:47:02.254+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'demo_dag_for_alerting_apis_final', 'check_previous_run_status', 'scheduled__2024-06-26T10:46:00+00:00', '--job-id', '4408', '--raw', '--subdir', 'DAGS_FOLDER/demo_dag_for_alerting_apis.py', '--cfg-path', '/tmp/tmpegihj9ul']
[2024-06-26T10:47:02.256+0000] {standard_task_runner.py:91} INFO - Job 4408: Subtask check_previous_run_status
[2024-06-26T10:47:02.322+0000] {task_command.py:426} INFO - Running <TaskInstance: demo_dag_for_alerting_apis_final.check_previous_run_status scheduled__2024-06-26T10:46:00+00:00 [running]> on host ced2311e92d2
[2024-06-26T10:47:02.446+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='demo_dag_for_alerting_apis_final' AIRFLOW_CTX_TASK_ID='check_previous_run_status' AIRFLOW_CTX_EXECUTION_DATE='2024-06-26T10:46:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-06-26T10:46:00+00:00'
[2024-06-26T10:47:02.447+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-06-26T10:47:02.463+0000] {baseoperator.py:399} WARNING - BranchPythonOperator.execute cannot be called outside TaskInstance!
[2024-06-26T10:47:02.463+0000] {python.py:237} INFO - Done. Returned value was: skip_run
[2024-06-26T10:47:02.464+0000] {branch.py:36} INFO - Branch into skip_run
[2024-06-26T10:47:02.464+0000] {skipmixin.py:178} INFO - Following branch skip_run
[2024-06-26T10:47:02.481+0000] {skipmixin.py:246} INFO - Skipping tasks [('start_run', -1)]
[2024-06-26T10:47:02.511+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-06-26T10:47:02.540+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=demo_dag_for_alerting_apis_final, task_id=check_previous_run_status, run_id=scheduled__2024-06-26T10:46:00+00:00, execution_date=20240626T104600, start_date=20240626T104702, end_date=20240626T104702
[2024-06-26T10:47:02.589+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-06-26T10:47:02.628+0000] {taskinstance.py:3498} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-06-26T10:47:02.633+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
